packages/gatsby-theme-carbon/src/components/markdown/Markdown.module.scss

```
@@ -21,13 +21,18 @@
 .paragraph--responsive {
   padding-left: $spacing-05;
 
+  @include carbon--breakpoint-down('sm') {
+    padding-right: $spacing-05;
+  }
+
   // 8 col
-  @include carbon--breakpoint('lg') {
-    width: 66.667%;
+  @include carbon--breakpoint('md') {
+    width: 75%;
   }
 
-  @include carbon--breakpoint-down('sm') {
-    padding-right: $spacing-05;
+  // 16 col
```

```suggestion
  // 8 col
```

This would still be 8 right?